webviews/common/createContext.ts

```diff
@@ -17,16 +17,31 @@ const defaultCreateParams: CreateParams = {
 };
 
 export class CreatePRContext {
+	public createParams: CreateParams;
+
 	constructor(
-		public createParams: CreateParams = { ...defaultCreateParams, ...vscode.getState() },
 		public onchange: ((ctx: CreateParams) => void) | null = null,
 		private _handler: MessageHandler | null = null,
 	) {
+		this.createParams = vscode.getState();
 		if (!_handler) {
 			this._handler = getMessageHandler(this.handleMessage);
 		}
 	}
 
+	get initialized(): boolean {
+		if (this.createParams.availableBaseRemotes.length !== 0
```

Comment:
Are the `!== 0` checks there because the variables are nullable?

Reply:
They're arrays and if the length is 0 then then `this.createParams` hasn't been initialized yet.